react/docs/vscode-extensions.md

```diff
@@ -0,0 +1,45 @@
+# VSCode Extensions
+
+## Introduction
+
+This guide intends to be a guide to useful React/React-Native extensions used along with VSCode. Those extensions are useful to have a productive working enviroment. Also, it will teach how to import/export those extensions.
```

Comment:
`This guide intends to be a guide` sounds kinda repetitive, i'd change it to something like `This guide will help you install some useful React/React-Native on your VSCode. ...`